internal/handlers/createlink.go

```diff
@@ -19,11 +20,17 @@ func (s *Server) CreateLink(writer http.ResponseWriter, request *http.Request) {
 		http.Error(writer, "Invalid request", http.StatusBadRequest)
 		return
 	}
+	cookie, err := request.Cookie("shortener_session")
```

Comment:
```suggestion
	userSession, err := request.Cookie("shortener_session")
```
Когда достаем конкретную куку она уже для нас не абстрактная cookie, а что-то конкретное